notebooks/02_dj_webinar.ipynb

```diff
@@ -0,0 +1,1403 @@
+{
```

Comment:
<p>Actually DataJoint no longer officially refers to the diagrams as ERD anymore. Hence for consistency, we could stop referring to ERD in our workshop as well. Perhaps calling it the "relationship diagram" would be a fair bet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>